src/main/java/core/basesyntax/MainApp.java

```diff
@@ -1,5 +1,17 @@
 package core.basesyntax;
 
 public class MainApp {
+    public static void main(String[] args) {
+        Machine bmw = new Excavator();
+        Machine volkswagen = new Bulldozer();
+        Machine audi = new Truck();
```

Comment:
```suggestion
        Machine excavator = new Excavator();
        Machine bulldozer = new Bulldozer();
        Machine truck = new Truck();
```

let's use more informative variable names)